src/main/java/com/eternalcode/parcellockers/locker/repository/LockerWrapper.java

```diff
@@ -0,0 +1,39 @@
+package com.eternalcode.parcellockers.locker.repository;
+
+import com.eternalcode.parcellockers.database.persister.PositionPersister;
+import com.eternalcode.parcellockers.locker.Locker;
+import com.eternalcode.parcellockers.shared.Position;
+import com.j256.ormlite.field.DatabaseField;
+import com.j256.ormlite.table.DatabaseTable;
+
+import java.util.UUID;
+
+@DatabaseTable(tableName = "lockers")
+class LockerWrapper {
+
+    @DatabaseField(id = true, columnName = "uuid")
+    private UUID uuid;
+
+    @DatabaseField(columnName = "description")
+    private String description;
+
+    @DatabaseField(columnName = "position", persisterClass = PositionPersister.class, unique = true)
```

Comment:
robisz wyszukiwanie po tej kolumnie, czy jest na niej index? Zobacz w DB jeśli nie to zobacz w tej adnotacji